src/driver.rs

```diff
@@ -60,9 +60,55 @@ fn test_arg_value() {
     assert_eq!(arg_value(args, "--bar", |p| p == "foo"), None);
     assert_eq!(arg_value(args, "--foobar", |p| p == "foo"), None);
     assert_eq!(arg_value(args, "--foobar", |p| p == "123"), Some("123"));
+    assert_eq!(arg_value(args, "--foobar", |p| p.contains("12")), Some("123"));
     assert_eq!(arg_value(args, "--foo", |_| true), None);
 }
 
+/// This function checks if the arguments contain a `--force-warn` for any clippy lint. This
+/// is different from [`arg_value`] as `--force-warn` can be present more than once.
```

Comment:
Maybe it shouldn't, but `arg_value` does check multiple args from left to right until one returns `true` so I think you could use it here. Found this out wondering why `--cap-lints=allow --cap-lints=warn` didn't force enable clippy

Reply:
Ohh, you're right! Thank you for pointing that out. Then this change became even simpler ^^